swift/Tests/Keystore/KeystoreKeyTests.swift

```diff
@@ -119,4 +119,25 @@ class KeystoreKeyTests: XCTestCase {
         let data = keystore.decryptPrivateKey(password: password)
         XCTAssertEqual(data?.hexString, "4357b2f9a6150ba969bc52f01c98cce5313595fe49f2d08303759c73e5c7a46c")
     }
+
+    struct KdfParams: Decodable {
+        let dklen: Int
+        let n: Int
+    }
+
+    struct EncryptionParameters: Decodable {
+        let kdf: String
+        let kdfparams: KdfParams
+    }
+
+    func testEncryptionParameters() {
```

Comment:
can you show an example how to encrypt with specific params?

Reply:
I have checked into this, and currently this is not possible.  Always default encryption parameters are used, and there is no way to influence that.
If this is needed, let's clarify and create a separate issue.